doc/man1/openssl-ca.pod.in

```diff
@@ -442,7 +442,8 @@ CA private key. Mandatory.
 =item B<RANDFILE>
 
 At startup the specified file is loaded into the random number generator,
-and at exit 256 bytes will be written to it.
+and at exit 256 bytes will be written to it. (Note: Using a RANDFILE is
+not necessary anymore, see the L</HISTORY> section.
```

Comment:
@levitte is this the correct way to reference sections in pod files? In other places, I see a `B<..>` around the reference:

https://github.com/openssl/openssl/blob/174cba8f87e08cf2324204ed3390ec156d314f77/doc/man1/openssl-ca.pod.in#L112-L113

Reply:
Yes, that's the correct way to do it (I assume you mean within the same POD file).
The references done with B\<...\> are incorrect (that's technical depth, we were not as aware of POD markup back a number of years)